detekt.yaml

```diff
@@ -1,5 +1,7 @@
 # See https://github.com/detekt/detekt/blob/master/detekt-core/src/main/resources/default-detekt-config.yml
 # for default settings.
+build:
+  excludeCorrectable: false
```

Comment:
Build will still fail on auto-correctable issues. This is because customizing local builds to pass while having CI builds pass wasn't trivial given how our build is set up now (I'll bring this up on salck)